src/users.ts

```diff
@@ -122,14 +122,21 @@ function getTeam(team: string, org: string, teams: Team[]) {
   });
 }
 
-export async function reconcileUsers() {
-  const promises = new Array<Promise<Octokit.AnyResponse | void>>();
+export async function reconcileUsers(reconcileAdmin = false) {
   const teams = await reconcileTeams();
```

Comment:
@JustinBeckwith it didn't look like `reconcileTeams` actually creates teams does it? Is this something we need to do manually right now?

Reply:
Going to be honest, I wrote this so long ago, I forget